tests/juniper_qfx_copper/juniper_qfx_copper_protocol_test.py

```diff
@@ -1148,10 +1149,9 @@ def test_assign_port_to_aggregated_interface(self):
         assert_that(ge001.xpath("ether-options/*"), has_length(1))
         assert_that(ge001.xpath("ether-options/speed/ethernet-10g"), has_length(1))
 
-        ge002 = self.get_interface("ge-0/0/2")
-        assert_that(ge002.xpath("*"), has_length(1))
-        assert_that(ge002.xpath("unit"), has_length(0))
-        assert_that(ge002.xpath("ether-options"), has_length(0))
+        with self.assertRaises(Exception) as exc:
+            self.get_interface("ge-0/0/2", )
```

Comment:
Real code does not raise an exception. I think this should check the state of the xml.